app/models/oral_history_content/interviewee_death.rb

```diff
@@ -0,0 +1,14 @@
+require 'attr_json'
+class OralHistoryContent
+  class IntervieweeDeath
+    include AttrJson::Model
+
+    validates_format_of :date, with: /\A\d{4}(-\d{2}(-\d{2})?)?\z/,
+      message: "must be of format YYYY[-MM-DD]"
+
+    # TODO validate `category` string can be either 'birth' or 'death'
```

Comment:
Let's do that TODO before merging... although wait, I think the TODO is no longer relevant, no need for a "category", instead you just have attributes `interviewee_birth` and `interviewee_death`. The way you've done it is great, so just eliminate old wrong TODO comment. 

    validates :exhibition, array_inclusion: { in: ControlledLists::EXHIBITION  }